### DISCUSSION TITLE
ActiveStorage?

### DISCUSSION THREAD

**Discussion by jaredmoody** [blackcandy-org/blackcandy#133] [Q&A] [answered]
Are you interested in replacing carrierwave with ActiveStorage? It seems to me it would be nice to remove the dependency and use what's built in to Rails. I could help with this migration.

**Comment by aidewoode** [comment, marked as answer]
OK, It's great if black candy can replacing carrierwave with ActiveStorage. Black candy use ActiveStorage from the beginning. But at that time ActiveStorage didn't support public access, this cause some performance issues. So I switch to carrierwave. And now ActiveStorage already support public access, so I think switch back to ActiveStorage is a good idea.
